### ISSUE TITLE
[BUG]Compatibility with the new version of Dart

### ISSUE THREAD

**Issue by lavara123** [jhomlala/betterplayer#260]
**Is your feature request related to a problem? Please describe.**
There are incompatibilities with the new version of the dart

> /better_player-0.0.49/lib/src/core/better_player.dart

`175   resizeToAvoidBottomPadding: false,`

this is an old way

**Describe the solution you'd like**
should be replaced with the new method:
[https://api.flutter.dev/flutter/material/Scaffold/resizeToAvoidBottomPadding.html](https://api.flutter.dev/flutter/material/Scaffold/resizeToAvoidBottomPadding.html)

**Other warnings**
```bash
opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:1111:7: Info: Struct 'ENUMLOGFONTEX' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class ENUMLOGFONTEX extends Struct {                                    
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:2835:7: Info: Struct 'BLUETOOTH_PIN_INFO' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class BLUETOOTH_PIN_INFO extends Struct {                               
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:2960:7: Info: Struct 'EXCEPINFO' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class EXCEPINFO extends Struct {}                                       
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:2966:7: Info: Struct 'PROPERTYKEY' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class PROPERTYKEY extends Struct {}                                     
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:2973:7: Info: Struct 'PROPVARIANT' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class PROPVARIANT extends Struct {}                                     
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:2978:7: Info: Struct 'SAFEARRAY' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class SAFEARRAY extends Struct {}                                       
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:2985:7: Info: Struct 'CLSID' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class CLSID extends Struct {}                                           
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:2992:7: Info: Struct 'STATSTG' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class STATSTG extends Struct {}                                         
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart:2999:7: Info: Struct 'NLM_SIMULATED_PROFILE_INFO' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class NLM_SIMULATED_PROFILE_INFO extends Struct {}                      
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/ffi-0.1.3/lib/src/utf8.dart:23:7: Info: Struct 'Utf8' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class Utf8 extends Struct {                                             
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/ffi-0.1.3/lib/src/utf16.dart:16:7: Info: Struct 'Utf16' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class Utf16 extends Struct {                                            
      ^                                                                 
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/ffi-0.1.3/lib/src/allocation.dart:47:33: Info: Support for using non-constant type arguments 'T' in this FFI API is deprecated and will be removed in the next stable version of Dart. Rewrite the code to ensure that type arguments are compile time constants referring to a valid native type.
  final int totalSize = count * sizeOf<T>();                            
                                ^                                       
Running Gradle task 'assembleRelease'...                                

```
![Ekran görüntüsü_2021-01-31_23-16-32](https://user-images.githubusercontent.com/12005386/106396654-4462c280-6401-11eb-9c28-4f4776314b3e.png)

**Comment by jhomlala** [comment]
`resizeToAvoidBottomPadding` will be solved with next version, but other errors can't be solved in Better Player. Are you using stable version of flutter?

**Comment by lavara123** [comment]
> `resizeToAvoidBottomPadding` will be solved with next version, but other errors can't be solved in Better Player. Are you using stable version of flutter?

`• Flutter version 1.26.0-18.0.pre.12`


Is it because it's a dev version ?

**Comment by jhomlala** [comment]
Yes exactly. Better Player is building around stable Flutter version. Using dev/beta version may lead to problems like above.

**Comment by jhomlala** [comment]
Fixed in 0.0.50.
